wrfhydropy/core/job.py

```diff
@@ -307,8 +333,27 @@ def _set_hrldas_times(self):
 
                 self._hrldas_times['noahlsm_offline']['restart_filename_requested'] = lsm_restart_file
 
-            self._hrldas_times['noahlsm_offline']['restart_frequency_hours'] = self.restart_freq_hr
-            self._hrldas_times['noahlsm_offline']['output_timestep'] = self.output_freq_hr * 3600
+            # TODO(JLM): I dont love this if statement, it's a bit hacky.
+            # Some of the tests call _set_hrldas/hydro_times when no namelist has been set.
```

Comment:
Should we just refactor the tests then?

Reply:
Let's remove the if statement and see why the test is failing and what the test is. Then we can make a better decision if this `if` statement is good or if a warning or error should be thrown.